proposals/NNNN-set-language-separately.md

```diff
@@ -0,0 +1,55 @@
+# Set language separately
+
+* Proposal: [SDL-NNNN](NNNN-set-language-separately.md)
+* Author: [zhouxin627](https://github.com/zhouxin627)

```

Comment:
Can you please include your name here?

Reply:
@theresalech Thanks, fixed in [7e8d6ec](https://github.com/smartdevicelink/sdl_evolution/pull/929/commits/7e8d6ec4da4bcbf1ef2512c946d8870da7792adc)